docs/docs/forms.md

```diff
@@ -114,8 +115,6 @@ class Form extends React.Component {
     return (
       <div>
         <input type="text"
-          placeholder="Hello!"
-          value={this.state.value}
```

Comment:
Can you please add line highlighting for the relevant lines in these examples? They're pretty long, and it would help to highlight whatever parts or properties that are being explained.